diff --git a/shell.c b/shell.c
index 426d589..fa64cf5 100644
--- a/shell.c
+++ b/shell.c
@@ -7282,7 +7282,7 @@ static int fsdirRegister(sqlite3 *db){
 #ifdef _WIN32
 
 #endif
-int sqlite3_fileio_init(
+int sqlite3_fileio_init_default(
   sqlite3 *db, 
   char **pzErrMsg, 
   const sqlite3_api_routines *pApi
@@ -21281,7 +21281,7 @@ static void open_db(ShellState *p, int openFlags){
     sqlite3_ieee_init(p->db, 0, 0);
     sqlite3_series_init(p->db, 0, 0);
 #ifndef SQLITE_SHELL_FIDDLE
-    sqlite3_fileio_init(p->db, 0, 0);
+    sqlite3_fileio_init_default(p->db, 0, 0);
     sqlite3_completion_init(p->db, 0, 0);
 #endif
 #ifdef SQLITE_HAVE_ZLIB
@@ -23436,7 +23436,7 @@ static int arDotCommand(
         );
         goto end_ar_command;
       }
-      sqlite3_fileio_init(cmd.db, 0, 0);
+      sqlite3_fileio_init_default(cmd.db, 0, 0);
       sqlite3_sqlar_init(cmd.db, 0, 0);
       sqlite3_create_function(cmd.db, "shell_putsnl", 1, SQLITE_UTF8, cmd.p,
                               shellPutsFunc, 0, 0);
